diff --git a/node_modules/react-native-push-notification/android/build.gradle b/node_modules/react-native-push-notification/android/build.gradle
index 2d3f075..5b8ee86 100644
--- a/node_modules/react-native-push-notification/android/build.gradle
+++ b/node_modules/react-native-push-notification/android/build.gradle
@@ -44,7 +44,7 @@ android {
 
 dependencies {
     def supportLibVersion = safeExtGet('supportLibVersion', '27.1.1')
-    def googlePlayServicesVersion = safeExtGet('googlePlayServicesVersion', '+')
+    def googlePlayServicesVersion = safeExtGet('googlePlayServicesVersion', '17.0.0')
     def firebaseVersion = safeExtGet('firebaseVersion', '+')
 
     implementation fileTree(dir: 'libs', include: ['*.jar'])
